index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <link rel="stylesheet" type="text/css" href="./css/projects.css">
+  <link rel="stylesheet" type="text/css" href="./css/about.css">
+  <link rel="stylesheet" type="text/css" href="./css/form.css">
+  <title>Personal-Profolio</title>
+
+</head>
+
+<body>
+  <!-- Headline Section Starts Here -->
+
+  <header class="navigation">
+    <div class="logo">
+      <h2>My logo</h2>
+    </div>
+    <nav class="navlinks">
+      <img src="/Img/Hlogo.png">
+    </nav>
+  </header>
+  <section class="headline-main">
+    <div class="headline">
+      <h1 class="headline-heading heading-large">I'm Mariane <br> Glad to see you!</h1>
+      <p class="headline-paragraph paragraph">I’m a software developer! I can help you build a product , feature
+        or website
+        Look
+        through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+      <p class="headline-sm-suport">LET’S CONNECT</p>
+      <ul class="social-icons">
```

Comment:
Kindly make the subtitle `LET'S CONNECT` a header tag `hx` (examples: `<h2>`, `<h3>`, etc...)`.

* This way will be semantically correct and the screen readers and other technologies will thank you. 😎